buildkite/data_source_team.go

```diff
@@ -46,10 +47,13 @@ func (t *teamDatasource) Metadata(ctx context.Context, req datasource.MetadataRe
 
 func (t *teamDatasource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
 	resp.Schema = schema.Schema{
-		MarkdownDescription: "This datasource allows you to get a team from Buildkite.",
+		MarkdownDescription: heredoc.Doc(`
+			Use this data source to retrieve a team by slug or id. You can find out more about clusters in the Buildkite
+			[documentation](https://buildkite.com/docs/pipelines/permissions).
```

Comment:
This docs link references clusters when this is around teams? 😉

Reply:
Lol yep good eyes!

Reply:
Honestly it was a tough find 👍